modules/dashboard/widgets-container.tf

```diff
@@ -134,3 +134,42 @@ module "container_network_error_widget" {
   pod       = each.value.pod
   namespace = each.value.namespace
 }
+
+module "container_volume_capacity_widget" {
+  source = "./modules/widgets/container/volume-capacity"
+
+  for_each = { for index, item in try(local.widget_config["container/volume-capacity"], []) : index => item }
+
+  datasource_uid = try(each.value.datasource_uid, null)
+  coordinates    = each.value.coordinates
+  period         = each.value.period
+  pvc_name       = each.value.pvc_name
+  container      = each.value.container
+  namespace      = each.value.namespace
+}
+
+module "container_volume_IOPS_widget" {
+  source = "./modules/widgets/container/volume-IOPS"
```

Comment:
lets keep naming for files and module names in lowercase event if the name is abbreviation like IOPS, so that here we have this
```
module "container_volume_iops_widget" {
  source = "./modules/widgets/container/volume-iops"
```